compiler/crates/relay-lsp/src/find_field_usages/find_field_locations.rs

```diff
@@ -5,21 +5,25 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+use std::sync::Arc;
+
 use common::Location as IRLocation;
 use graphql_ir::Program;
 use graphql_ir::Visitor;
 use intern::string_key::StringKey;
+use schema::SDLSchema;
 use schema::Schema;
 
 use crate::find_field_usages::FieldUsageFinder;
 
 pub(crate) fn find_field_locations(
     program: &Program,
+    schema: &Arc<SDLSchema>,
```

Comment:
Given your comment, it sounds like the presence of the schema here is to work around a larger bug/issue of the program.schema being stale. Could we add a comment here clarifying that? That way if/when we resolve that issue we can switch back to the simpler approach of using `program.schema`.

Reply:
I have actually reverted the change and disabled Relay Resolver field renaming for now - it was still buggy. I've raised an issue to track it: https://github.com/facebook/relay/issues/4447